tests/jenkins/jobs/uploadSnapshotsToS3_Jenkinsfile.txt

```diff
@@ -0,0 +1,15 @@
+   uploadSnapshotsToS3_Jenkinsfile.run()
+      uploadSnapshotsToS3_Jenkinsfile.pipeline(groovy.lang.Closure)
+         uploadSnapshotsToS3_Jenkinsfile.echo(Executing on agent [label:none])
+         uploadSnapshotsToS3_Jenkinsfile.stage(uploadMinSnapshotsToS3, groovy.lang.Closure)
+            uploadSnapshotsToS3_Jenkinsfile.script(groovy.lang.Closure)
+               uploadSnapshotsToS3_Jenkinsfile.uploadMinSnapshotsToS3({manifest=tests/jenkins/data/opensearch-1.3.0.yml})
+                  uploadMinSnapshotsToS3.legacySCM(groovy.lang.Closure)
+                  uploadMinSnapshotsToS3.library({identifier=jenkins@20211123, retriever=null})
+                  uploadMinSnapshotsToS3.readYaml({file=tests/jenkins/data/opensearch-1.3.0.yml})
+                  InputManifest.asBoolean()
+                  uploadMinSnapshotsToS3.echo(Retreving build manifest from: workspace/builds/opensearch/manifest.yml)
+                  uploadMinSnapshotsToS3.readYaml({file=workspace/builds/opensearch/manifest.yml})
+                  BuildManifest.asBoolean()
+                  uploadMinSnapshotsToS3.withAWS({role=dummy_role, roleAccount=1234, duration=900, roleSessionName=jenkins-session}, groovy.lang.Closure)
+                     uploadMinSnapshotsToS3.s3Upload({file=workspace/builds/opensearch/dist/, bucket=dummy_bucket, path=snapshots/core/opensearch/1.1.0/opensearch-min-1.1.0-linux-x64-latest.tar.gz})
```

Comment:
The version here appears to be a non-snapshot version because the build manifest it is referring to is non-snapshot. 
Should I commit a snapshot build manifest file so that this appears correctly here? eg:
```
build:
  architecture: arm64
  id: '29'
  name: OpenSearch
  platform: linux
  version: 1.2.2-SNAPSHOT
```

Reply:
Yes

Reply:
Added snapshot manifest. Might be useful for other snapshot scenarios in future.